group_vars/docker-registry/storage_backend

```diff
@@ -1,17 +0,0 @@
----
-# default storage type is local (file).
-# storage_type: "local"
-
-# storage_type: "gcs"
-gcs_bucket: "gds-docker-registry"
-gcs_storage_path: "/"
-gcs_oauth2: false
-gcs_access_key: "{{ lookup('env', 'GCS_ACCESS_KEY') }}"
-gcs_secret_key: "{{ lookup('env', 'GCS_SECRET_KEY') }}"
-
-# storage_type: "s3"
-s3_region: "eu-west-1"
-s3_bucket: "docker-registry-store"
-s3_storage_path: "/srv/docker"
```

Comment:
This variable may need to be added as well if we want to support multiple clusters in the same bucket. By default registry folders are created at top level in the bucket.